apps/pie-docs/src/accessibility/testing-for-accessibility.md

```diff
@@ -0,0 +1,135 @@
+
+We offer guidance on the different types of accessibility testing that should be used in your projects. We recommend a **mixture of automated, manual, and user testing  methods** to ensure that most accessibility issues can be detected.
+
+{% contentPageImage {
+    src:"../../../assets/img/accessibility/testing-for-accessibility/aa-wcag.svg",
+    mobileSrc:"../../../assets/img/accessibility/testing-for-accessibility/aa-wcag_narrow.svg",
+    caption: "Our products should meet the AA level of WCAG 2.1’s accessibility guidelines."
+} %}
+
+---
+
+## Automated testing
+
+Automated testing tools can audit your product against the WCAG to ensure that you're following best practices that can be detected.
+
+We estimate that **automated testing can detect a great amount of common WCAG issues**. Automated testing also gives your team more time to focus on the more complex accessibility tests that need to be completed manually.
+
+We recommend using tools such as Deque’s [axe-core](https://www.deque.com/axe/) for the web, [Google’s Toolbox for Accessibility for iOS](https://github.com/google/GTXiLib) (GTXiLib), or [Google’s Accessibility Test Framework for Android](https://github.com/google/Accessibility-Test-Framework-for-Android) (ATF) for automated testing.
+
+{% notification {
+  type: "information",
+  message: "Please [reach out to us](/support/contact-us/) if you need help integrating these tools."
+} %}
+
+It’s important to note that whilst automated accessibility tests are a great step towards providing an accessible application, **they do not guarantee that your app is accessible to real users**.
+
+Some of the issues that automatic checks don’t usually cover are:
+- Correct reading order of content
+- Appropriate use of HTML
+- Good keyboard navigation experience
+- Useful image captions and alt text
+- Cognitive accessibility
+
+{% notification {
+  type: "warning",
+  message: "Automated tests should always be used in combination with a variety of manual test methods."
+} %}
+
+---
+
+## Manual testing
+
+For manual accessibility testing, we recommend referring to a comprehensive [accessibility checklist](https://www.a11yproject.com/checklist/) for web products to help you test for common accessibility barriers.
+
+**The A11y Project checklist** provides an excellent resource that includes items such as checking for proper colour contrast, keyboard accessibility, and the correct use of descriptive text and alt attributes for images. This checklist can serve as a good foundation for testing the accessibility of your product.
+
+{% notification {
+  type: "information",
+  message: "You can use The A11y Project’s [accessibility checklist](https://www.a11yproject.com/checklist/) as a main resource to manual test your products."
```

Comment:
```suggestion
  message: "You can use The A11y Project’s [accessibility checklist](https://www.a11yproject.com/checklist/) as a main resource to manually test your products."
```

Reply:
@jordifornellsJET

Reply:
Let's make this change (I'll update Figma)